exchange/exchange.go

```diff
@@ -102,6 +102,7 @@ type AuctionRequest struct {
 	Account     config.Account
 	UserSyncs   IdFetcher
 	RequestType pbsmetrics.RequestType
+	Timestamp   *time.Time
```

Comment:
Yay! This is proving useful already.

1. Do you think naming it 'StartTime' would be an improvement?
2. Is there a particular reason you're making this a pointer? Doesn't seem necessary IMHO.

Reply:
The naming was after the json attribute `auctiontimestamp` and event parameter `ts` where this would be used. I agree that `ReceivedTime` is a more apt description. Should I change it both here and in analytics ?

As for the pointer - I didn't want a copy with the source of truth elsewhere. Incidentally, the nil test also avoided breaking all the tests by returning `auctiontimestamp: 0`. It's also a minor size optimization, and I was thinking maybe Account should also be a reference.

Reply:
Hmm, the docs for time.Time say
> Programs using times should typically store and pass them as values, not pointers. That is, time variables and struct fields should be of type time.Time, not *time.Time.
so I will change it.

Reply:
> I agree that ReceivedTime is a more apt description (edit: but StartTime sounds better).

I was on the fence there too.

> Should I change it both here and in analytics ?

Yes please. I think we should try to keep consistent. To be clear though, let's keep the "auctiontimestamp" json name for compatibility with PBS-Java.

> It's also a minor size optimization

In Go, a pointer forces a heap allocation whereas a value will avoid it and use the stack. At some point the value copy operation can take a long time, but for smaller objects pass by value is typically preferred. A notable exception being JSON serialization which forces the use of a pointer to determine default vs not specified... and that bugs me to no end. Grrr.